sessions/react-effects-and-localstorage/journal-app-localstorage/src/components/EntriesSection/index.js

```diff
@@ -0,0 +1,46 @@
+import "./EntriesSection.css";
+import Divider from "../Divider";
+import Entry from "../Entry";
+import Tabs from "../Tabs";
+import Tab from "../Tab";
+import Badge from "../Badge";
+import { Fragment } from "react";
+
+export default function EntriesSection({
+  entries,
+  onToggleFavorite,
+  showFavorites,
+  amountAllEntries,
+  amountFavoriteEntries,
+  onShowAllEntries,
+  onShowFavoriteEntries,
+}) {
+  return (
+    <section className="entries-section">
+      <Tabs>
+        <Tab onClick={onShowAllEntries} active={!showFavorites}>
```

Comment:
I see why we do `active={!showFavorites}` here, but it was hard to understand in the first place; just haven't seen this so far, I think.